tests/wsl/wsl_cmd_check.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Validate WSL image from host
+# Maintainer: qa-c <qa-c@suse.de>
+
+use Mojo::Base qw(windowsbasetest);
+use testapi qw(assert_and_click type_string get_var);
+use version_utils qw(is_sle is_opensuse);
+
+my %expected = (
+        provider => get_var('WSL2') ? 'microsoft' : '(wsl|kvm)',
+        mount => '/mnt/c'
+);
+
+sub run {
+    my $self = shift;
+
+    assert_and_click 'powershell-as-admin-window';
+    type_string "exit\n";
+    $self->open_powershell_as_admin();
+    $self->run_in_powershell(cmd => 'wsl --list --verbose');
+    $self->run_in_powershell(cmd => "wsl mount | Select-String -Pattern $expected{mount}");
+    $self->run_in_powershell(cmd => qq{wsl ls $expected{mount}});
+    $self->run_in_powershell(cmd => qq/wsl systemd-detect-virt | Select-String -Pattern "$expected{provider}"/);
+    $self->run_in_powershell(cmd => 'wsl /bin/bash -c "dmesg | head -n 20"');
+    $self->run_in_powershell(cmd => 'wsl env');
+    $self->run_in_powershell(cmd => 'wsl locale');
+    $self->run_in_powershell(cmd => 'wsl date');
+    if (is_opensuse || (is_sle && get_var('SCC_REGISTER') =~ m/'yast'/i)) {
```

Comment:
WDYT to move `is_sut_reg` (possibly) to `version_utils` and reuse the subroutine here?

Reply:
The module was initially just my locally used sketch, however as we have no serial console in WSL2, it actually becomes quite handy to keep this module for a while. 
I do not think it is robust enough to be used in a generic lib. We can think about that idea and push it in another PR. :)